kubectl-rook-ceph.sh

```diff
@@ -179,6 +193,50 @@ function run_operator_restart_command () {
   kubectl --namespace "$NAMESPACE" rollout restart deploy/rook-ceph-operator
 }
 
+function path_cm_rook_ceph_operator_config() {
+  if [[ "$#" -ne 2 ]]; then
+    fail_error "require exactly 2 subcommand: $*"
+  fi
+  kubectl --namespace "$NAMESPACE" patch configmaps rook-ceph-operator-config --type json --patch  "[{ op: replace, path: /data/$1, value: $2 }]"
+}
+
+function get_mon_endpoints (){
+  end_of_command_parsing "$@" # end of command tree
+  kubectl --namespace "$NAMESPACE" get cm rook-ceph-mon-endpoints -o json | jq --monochrome-output '.data.data'| tr -d '"'
```

Comment:
This returns `a=10.110.30.112:6789` and we don't need `a=`, we just want the IP address:port.

Reply:
in this case, we have single mon but once we have more than one more it will be confusing I believe.

Reply:
Just remove `=` and any `[a-z]` from the output so we should get a proper list.

Reply:
What will the output of multiple mons look like? Each on its own line?

Reply:
now this what it looks like 
``` 
kubectl get cm rook-ceph-mon-endpoints -o json | jq --monochrome-output '.data.data'| tr -d '"' | tr -d '='|sed 's/[A-Za-z]*//g'
10.96.164.82:6789,10.108.197.247:6789,10.111.242.191:6789
```